apps/astarte_pairing/lib/astarte_pairing_web/controllers/fdo_onboarding_controller.ex

```diff
@@ -55,4 +55,31 @@ defmodule Astarte.PairingWeb.FDOOnboardingController do
         |> send_resp(400, err)
     end
   end
+
+  def prove_device(conn, _params) do
+    realm_name = conn.params["realm_name"]
+    cbor_body = conn.assigns[:cbor_body]
```

Comment:
nit: in general, always use hard failing mechanisms to avoid hard to detect bugs after a code refactor


```suggestion
    realm_name = Map.fetch!(conn.params, "realm_name")
    cbor_body = conn.assigns.cbor_body
```